src/grt/src/GlobalRouter.cpp

```diff
@@ -689,7 +689,6 @@ void GlobalRouter::updateDirtyNets(std::vector<Net*>& dirty_nets)
       dirty_nets.push_back(db_net_map_[db_net]);
       routes_[db_net].clear();
       db_net->clearGuides();
-      fastroute_->clearNetRoute(db_net);
```

Comment:
I removed it because when we add the net to the dirty list, the route of it will be removed later when we add it to the FastRouteCore. Tho I believe I'll restore it, just to ensure the resources will be released.